assets/js/base/components/cart-checkout/shipping-rates-control-package/index.tsx

```diff
@@ -34,21 +35,28 @@ interface Destination {
 	country: string;
 }
 
+export interface PackageData {
+	destination: Destination;
+	name: string;
+	// eslint-disable-next-line camelcase
+	shipping_rates: CartShippingRateItemShippingRate[];
+	items: PackageItem[];
+}
+
+export type PackageRateRenderOption = (
+	option: CartShippingRateItemShippingRate
+) => PackageRateOption;
+
 interface PackageProps {
-	packageId: string;
-	renderOption: ( option: Rate ) => PackageRateOption;
-	collapse: boolean;
-	packageData: {
-		destination: Destination;
-		name: string;
-		// eslint-disable-next-line camelcase
-		shipping_rates: Rate[];
-		items: PackageItem[];
-	};
+	/* PackageId can be a string, WooCommerce Subscriptions uses strings for example, but WooCommerce core uses numbers */
```

Comment:
Subscriptions may not be the only fwiw. Core allows strings but uses numbers internaly.

Reply:
Sure, thanks 😄 This was just an example of a time when they are strings explaining why it was typed as `string | number`